### DISCUSSION TITLE
请问，符号对照表和缩略语对照表，按照默认格式是在页面左对齐的，有办法能调整为页面居中吗？

### DISCUSSION THREAD

**Discussion by Zhaodimmmm** [note286/xduts#207] [Q&A] [answered]
请问，符号对照表和缩略语对照表，按照默认格式是在页面左对齐的，有办法能调整为页面居中吗？
![1723811494259](https://github.com/user-attachments/assets/5bd7aa0f-9330-49bd-89a1-8951fc2ff097)

**Comment by note286** [comment, marked as answer]
https://github.com/note286/xduts/discussions/40#discussioncomment-5084749
